SU2_CFD/src/solver_direct_mean.cpp

```diff
@@ -4325,8 +4322,9 @@ void CEulerSolver::SetInitialCondition(CGeometry **geometry, CSolver ***solver_c
    all the multigrid levels, this is important with the dual time strategy ---*/
   
   if (restart && (ExtIter == 0)) {
-    
     Solution = new su2double[nVar];
+		solver_container[0][FLOW_SOL]->Set_MPI_Solution(geometry[0], config);
```

Comment:
Let's leave this here for now, but I moved this chunk of code into the solver preprocessing part of the driver in the forthcoming changes with the restarts.